back-end/src/main/java/com/skynet/skymed/model/Pessoa.java

```diff
@@ -109,23 +134,32 @@ public Endereco getEndereco() {
 		return endereco;
 	}
 
-	public void setEndereco(Endereco endereco) {
+	public void setEndereco(Endereco endereco) throws Exception {
+		if (endereco == null) {
+			throw new Exception("Endereço inválido.");
+		}
 		this.endereco = endereco;
 	}
 
 	public String getTokenDeAutenticacao() {
 		return tokenDeAutenticacao;
 	}
 
-	public void setTokenDeAutenticacao(String tokenDeAutenticacao) {
+	public void setTokenDeAutenticacao(String tokenDeAutenticacao) throws Exception {
+		if (tokenDeAutenticacao.length() > 4 || tokenDeAutenticacao.length() < 4) {
```

Comment:
```suggestion
		if (tokenDeAutenticacao.length() > 4 ) {
```
Certeza q o token só tem 4 caracteres?

Reply:
absoluta